packages/core/integration-tests/test/javascript.js

```diff
@@ -339,6 +339,68 @@ describe('javascript', function() {
     assert.equal(await output(), 3);
   });
 
+  it('should prefetch bundles when declared as an import attribute statically', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/dynamic-static-prefetch/index.js'),
+      // TODO: This test fails nondeterministically with MemoryFS
```

Comment:
The MemoryFS nondeterministically throws when reading the file, and some of the tests that follow this seem to randomly fail once that's happened. Any ideas @devongovett?

Reply:
I'd have to run it to see. What kind of error do you get?